src/components/sign/Sign.js

```diff
@@ -23,7 +23,8 @@ class Sign extends Component {
     }
 
     handleAddFunds = () => {
```

Comment:
@Patrick1904 do you have any suggestion on what's best way to test this? See here how I tried to trigger:
https://github.com/nearprotocol/near-wallet/pull/467/commits/a4b610743bd4c6b8cde05ebed9f539cdb9d19538

Reply:
Not off the top of my head. Some bugs I fix could have been prevented with proptypes. Not this one though.

Reply:
@Patrick1904 I'm more concerned on how to actually trigger the related Add Funds UI (as I failed to do it) vs how to prevent bugs like this from happening.